config.yml

```diff
@@ -11,7 +11,7 @@ ETH:
   adm0c_bound: "ADM0_EN"
   adm1c_bound: "ADM1_EN"
   adm2c_bound: "ADM2_EN"
-  pop_filename: "eth_admpop_adm2_2020.csv"
+  pop_filename: "Population_OCHA_2020/eth_admpop_adm2_2020.csv"
```

Comment:
is there a URL to obtain that csv? can we have a comment that explains how to procure the csv? We might also need a broader methodology document explaining why we changed the source

Reply:
Good point, added the url link. Since this data (and admin boundaries) comes from HDX it might actually be neater to use the HDX-api? Then we will always have the most up-to-date versions

With why we changed the source, I am assuming you mean the admin boundaries, since we didn't change the subnational population data. If so, yes we should.